distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Value_Type_Helpers.enso

```diff
@@ -210,18 +210,29 @@ type Addition_Kind
     ## PRIVATE
     Text_Concat
 
+## Returns the operation kind based on types of the inputs.
+   If both inputs are Null, it is impossible to tell the kind, so Nothing is returned and the caller may decide what to do.
```

Comment:
```suggestion
   If both inputs are Null, it is impossible to tell the kind, so Nothing is returned and the caller may decide what to do.
   Otherwise, the kinds must be the same `Addition_Kind`, and that kind is returned.
```